dashboard/config/locales/en.yml

```diff
@@ -250,6 +250,7 @@ en:
       %{name}
   parent_mailer:
     student_associated_subject: 'Login information for Code.org'
+    parent_email_added_subject: 'Welcome to Code.org'  
```

Comment:
Can you add an exclamation point to the end of the subject? 'Welcome to Code.org!' (same as the teacher email)

Reply:
Done!